server/api/profile.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const { createProfile } = require("../services/database/profile");
+const router = express.Router();
+
+/** route is /profile/ (status is defined as prefix in index.js) */
+router.post("/", (req, res) => {
+  const db_newProfile = req.body;
+  createProfile(db_newProfile);
+  res.status(201).send("Profile created");
```

Comment:
We should use try() and catch() to catch any error. 
E.g
```
try {
  const db_newProfile = req.body;
  createProfile(db_newProfile);
  res.status(201).send("Profile created");
} catch (error) {
  res.status(500).send("Error Profile did not create");
}
```

Reply:
Done like this: 

> createProfile(db_newProfile).then((data) => {
> ```suggestion
>   res.status(201).send("Profile created");
> ```
> ```suggestion
>   res.status(201).send("Profile created");
> ```
> res.sendStatus(201);
> })
> .catch((err) => {
> console.error(err);
> res.sendStatus(500);
> });